index.yaml

```diff
@@ -3494,15 +3533,15 @@ entries:
       catalog.cattle.io/requires-gvr: networking.istio.io.virtualservice/v1beta1
     apiVersion: v1
     appVersion: 0.8.0
-    created: "2021-06-11T16:26:35.235675-07:00"
+    created: "2021-06-17T10:22:59.914906-07:00"
     description: The application deployment engine for Kubernetes
-    digest: d12341801bbe55648737273240485a98ac1da7d21a67d3a8ade45a82aa648528
+    digest: 8baa5c330cc152b3d3d87f918ed3ff96b927efad412f4b97bd7db90445e28602
     home: https://rio.io
     icon: https://charts.rancher.io/assets/logos/rio.svg
     name: rio
     urls:
-    - assets/rio/rio-0.8.1.tgz
-    version: 0.8.1
+    - assets/rio/rio-0.8.001.tgz
+    version: 0.8.001
```

Comment:
This is correct; the version should not have been 0.8.1 in the first place since we already released 0.8.001, even though that is invalid semver.

We are releasing a 0.8.1 in this release as a result.